src/main/java/Duke/DukeSave.java

```diff
@@ -0,0 +1,96 @@
+package Duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+//import java.nio.file.Files;
+//import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class DukeSave {
+    /**
+     * Divider used to separate details of a task saved in file
+     */
+    private static final String SAVEFILESEPARATOR = ", ";
```

Comment:
There is an issue in the naming convention here. Remember to use `_` to separate the words in the name. For example, `FILE_STORAGE`, `MAX_INT`.